src/apps/btc/btc.c

```diff
@@ -138,7 +138,7 @@ app_btc_result_t app_btc_address_multisig(
     }
 
     const char* title = "Receive to";
-    if (!apps_btc_confirm_multisig(title, coin, multisig_registered_name, multisig, false)) {
+    if (!apps_btc_confirm_multisig(title, coin, multisig_registered_name, multisig, false, 0)) {
```

Comment:
use `BTCRegisterScriptConfigRequest_XPubType_AUTO_ELECTRUM` for better readability? otherwise, it isn't clear what 0 means here.

Reply:
Also in btc_sign_validate.c where it calls `apps_btc_confirm_multisig(..., 0)`

Reply:
put it in a var called `ignore`, to indicate that it is ignored, since verify_xpubs is false.